fanuc_r2000ic_support/config/opw_parameters_r2000ic165f.yaml.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# Parameters for use with IK solvers which support OPW (Ortho-Parallel Wrist)
+# kinematic configurations, as described in the paper "An Analytical Solution
+# of the Inverse Kinematics Problem of Industrial Serial Manipulators with an
+# Ortho-parallel Basis and a Spherical Wrist" by Mathias Brandstötter, Arthur
+# Angerer, and Michael Hofbaur (Proceedings of the Austrian Robotics Workshop
+# 2014, 22-23 May, 2014, Linz, Austria).
+#
+# The moveit_opw_kinematics_plugin package provides such a solver.
+#
+opw_kinematics_geometric_parameters:
+  a1:  0.312
+  a2:  -0.225
+  b:   0.000
+  c1:  0.670
+  c2:  1.075
+  c3:  1.280
+  c4:  0.215
+opw_kinematics_joint_offsets: [0, 0, -1.5707963267949, 0, 0, 3.14159265358979]
```

Comment:
Oh, and the ROS parameter server supports a special yaml ctor which converts radians to degrees.

See [ros-industrial/fanuc/fanuc_lrmate200ib_support/config/opw_parameters_lrmate200ib.yaml](https://github.com/ros-industrial/fanuc/blob/bc4b8cb261aa519099c5c1e4fbdbc9ad12c85ce9/fanuc_lrmate200ib_support/config/opw_parameters_lrmate200ib.yaml#L19) for an example.